django_select2/static/django_select2/django_select2.js

```diff
@@ -8,11 +8,24 @@
         var settings = $.extend({
             ajax: {
                 data: function (params) {
-                    return {
+                    var res = {
```

Comment:
Why not move this to the bottom and return right away. I don't see you using this variable anywhere but in the return command.